install-steps.md

```diff
@@ -603,17 +609,17 @@ to fix the ConfigMap for the Metal3 operator. This ConfigMap is used to notify
       namespace: openshift-machine-api
     data:
       cache_url: ''
-      deploy_kernel_url: http://172.22.0.3:6180/images/ironic-python-agent.kernel
-      deploy_ramdisk_url: http://172.22.0.3:6180/images/ironic-python-agent.initramfs
+      deploy_kernel_url: http://<provisioning_ip>:6180/images/ironic-python-agent.kernel
+      deploy_ramdisk_url: http://<provisioning_ip>:6180/images/ironic-python-agent.initramfs
       dhcp_range: 172.22.0.10,172.22.0.100
       http_port: "6180"
-      ironic_endpoint: http://172.22.0.3:6385/v1/
+      ironic_endpoint: http://<provisioning_ip>:6385/v1/
       ironic_inspector_endpoint: http://172.22.0.3:5050/v1/
```

Comment:
Maybe we would want to remove the above IP as well.

Reply:
@gsr-shanks @dorzheh this looks like it doesn't belong to this PR. Another rebase against the master branch should align the files.